datadog_lambda/patch.py

```diff
@@ -17,15 +17,11 @@
     get_dd_trace_context,
     dd_tracing_enabled,
 )
+from collections.abc import MutableMapping
 
 logger = logging.getLogger(__name__)
 
-if sys.version_info >= (3, 0, 0):
-    httplib_module = "http.client"
-    from collections.abc import MutableMapping
-else:
-    httplib_module = "httplib"
-    from collections import MutableMapping
+httplib_module = "http.client"
```

Comment:
You could probably make this local to `_patch_httplib`. You might also want to consider changing that function's name and removing Python 2 from its docstring.